frontend/src/components/CollapseButton.tsx

```diff
@@ -36,24 +37,27 @@ const DefaultText = () => {
 const CollapseButton = ({
   children = <DefaultText />,
   expanded,
+  withoutBadge,
```

Comment:
Would prefer the option `showBadge` otherwise our code will be full of double negatives.